src/data/contributors/TejaswiniVakkalagaddi.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Tejaswini Vakkalagaddi",
+    "skills": ["Django", "Python", "Java Script", "HTML"],
+    "cause": "Code for Self Improvement and to empower Women",
```

Comment:
Code for is always the part of the string 
![image](https://user-images.githubusercontent.com/48255244/94792068-18edae00-03f6-11eb-8658-a5c70b809a0a.png)

Reply:
Hello Abhishek do I need create another pull request by modifying my code.